src/evergreen/performance_results.py

```diff
@@ -0,0 +1,69 @@
+# -*- encoding: utf-8 -*-
+"""Performance results representation of evergreen."""
+from __future__ import absolute_import
+
+from evergreen.base import _BaseEvergreenObject, evg_attrib, evg_datetime_attrib
+
+
+class PerformanceResult(_BaseEvergreenObject):
+    """Representation of a single performance result from evergreen"""
+    start = evg_datetime_attrib('start')
+    end = evg_datetime_attrib('end')
+    name = evg_attrib('name')
+    workload = evg_attrib('workload')
+    results = evg_attrib('results')
+
+    def __init__(self, json, api):
+        """Create an instance of performance result data."""
+        super(PerformanceResult, self).__init__(json, api)
+
+
+class PerformanceResultData(_BaseEvergreenObject):
+    """Representation of performance result data from evergreen."""
+
+    storage_engine = evg_attrib('storageEngine')
+    # Legacy support
+    storageEngine = evg_attrib('storageEngine')
```

Comment:
I don't this we should include both `storage_engine` and `storageEngine`. Users of this will have to make a change if they want to use this library. So I'm included to just support `storage_engine`.

Reply:
Will only use `storage_engine`